zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/OpensearchEngineClient.java

```diff
@@ -150,7 +158,53 @@ public void putSettings(
   }
 
   @Override
-  public void putIndexLifeCyclePolicy(final String policyName, final String deletionMinAge) {}
+  public void putIndexLifeCyclePolicy(final String policyName, final String deletionMinAge) {
+    final var request = createIndexStateManagementPolicy(policyName, deletionMinAge);
+
+    try (final var response = client.generic().execute(request)) {
```

Comment:
💭 I think this is a valid way around executing raw REST requests here. 
I know that operate had an `OpenSearchISMOperations` implementation with various methods to deal with ISM, would we need to port this here instead at some point?